client/me/help/chat-business-concierge-notice/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { identity } from 'lodash';
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import i18n, { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import HelpTeaserButton from '../help-teaser-button';
+import { isBusinessPlanUser } from 'state/selectors';
+
+class ChatBusinessConciergeNotice extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
+		isBusinessPlanUser: PropTypes.bool.isRequired,
+		from: PropTypes.string.isRequired,
+		to: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	trackCalendlyOfferClick = () => {
+		analytics.tracks.recordEvent( 'calypso_help_calendly_offer_click' );
+	};
+
+	render = () => {
+		const { translate } = this.props;
```

Comment:
Just nitpicking around, not a blocker but I guess we could destructure all used props here instead of using `this.props`